libtelemetry/src/main/java/com/mapbox/android/telemetry/CertificateBlacklist.java

```diff
@@ -223,20 +209,12 @@ private String determineConfigEndpoint() {
     return COM_CONFIG_ENDPOINT;
   }
 
-  private List<String> extractResponse(Response response) throws IOException {
-    String responseData = response.body().string();
-
-    Gson gson = new Gson();
-    JsonObject jsonObject = gson.fromJson(responseData, JsonObject.class);
-
-    Type listType = new TypeToken<List<String>>() {}.getType();
-    List revokedKeys = gson.fromJson(jsonObject.get("RevokedCertKeys"), listType);
-
-    return revokedKeys;
-  }
+  private void saveTimestamp() {
+    Date date = new Date();
```

Comment:
what's up with `Date` here vs. `System.currentTimeMillis()` on line 95? Should we use `Date` there as well?